0006-adopt-existing-clusters-for-kaas-management/0006-adopt-existing-clusters-for-kaas-management.md

```diff
@@ -0,0 +1,7 @@
+# Adopt existing clusters for KaaS management
```

Comment:
[There is an issue in CAPZ upstream](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/1173) to track the development for adopting `AKS` cluster specifically. Maybe we can link it to add context.